### PR TITLE
Update the browser list config and support documentation

### PR DESCRIPTION
**Pull request by jimmynotjim**
As part of the FEWD Browser Support Working Group we have been
re-evaluating what browsers we're supporting. We had agreed to use a
browserlistrc file to track our needs, but as we looked closer we
realized our build process is broken up by browser and would necessitate
a complex browserlistrc setup.

## Changes

- Converted browser config to a new separate config file.
- Reduced configs to only those necessary (many browsers are included in
  "last2").
- Added human readable browser support notes to contributing doc.

## Testing

- run `gulp build`

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### Browsers

- [ ] Chrome
- [ ] Safari
- [ ] FF
- [ ] IE10
- [ ] IE9
- [ ] IE8
- [ ] Opera
- [ ] iOS
- [ ] Android
- [ ] Blackberry Bold

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: config/browser-list-config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+];
+
+const LAST_2_IE_8_UP = [
+  'last 2 versions',
+  'Explorer >= 8'
+];
+
+module.exports = {
+  LAST_2_IE_9_UP,
+  LAST_2_IE_8_UP
```
Comment (Scotchester) [inline config/browser-list-config.js]: What do you think about renaming these to be extra clear? Something like, `LAST_2_PLUS_IE_8_AND_UP`. They're already a disaster to type without code completion, what's two more words?

**Comment by Scotchester** [comment]
We have a problem with the tests in Travis, though.

File: config/browser-list-config.js (hunk 1 above)
Comment (jimmynotjim) [reply config/browser-list-config.js]: Yeah, I guess it's not much worse now. I'll make that change here and in cfgov-refresh.

**Comment by jimmynotjim** [comment]
Snuck in some hopeful fixes to the release script

https://www.youtube.com/watch?v=-W6as8oVcuM

**Pull request merged**